### PR TITLE
Moving vmware alerts to a dedicated chart

### PR DESCRIPTION
**Pull request by richardtief**

### CODE REVIEW

File: prometheus-rules/prometheus-vmware-rules/values.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# The name of the Prometheus to which the alert- and aggregation rules will be assigned to.
+prometheusName: vmware
```
Comment (viennaa) [inline prometheus-rules/prometheus-vmware-rules/values.yaml:2]: What happens with the sharded prometheus setup here. Haven't checked the label yet, but i'd expect to have a label `prometheus: vmware-vc-a-0` or similar. or is this the in between approach, with the other PR upcoming to be merged after the sharded setup?

Comment (richardtief) [reply prometheus-rules/prometheus-vmware-rules/values.yaml:2]: This chart reflects the current state. We can easily extend it by looping through the same secrets file with the different targets.

Comment (richardtief) [reply prometheus-rules/prometheus-vmware-rules/values.yaml:2]: Yes.
> is this the in between approach, with the other PR upcoming to be merged after the sharded setup?

**Review by viennaa — APPROVED** [review]
Thanks for moving over the alerts. It's good from my pov, just opened a discussion in the respective team channel, if this folder is looped or not (hence every folder would need to reflect a cluster)

**Pull request merged**